lls/main.c

```diff
@@ -721,12 +742,25 @@ run_lls(struct net_config *net_conf, struct lls_config *lls_conf)
 		goto stage3;
 	}
 
+	/*
+	 * Do Gatekeeper log scan every gatekeeper_log_scan_interval_sec
+	 * seconds.
+	 */
+	rte_timer_init(&lls_conf->log_timer);
+	ret = rte_timer_reset(&lls_conf->log_timer,
+		gatekeeper_log_scan_interval_sec * rte_get_timer_hz(),
+		PERIODICAL, lls_conf->lcore_id, log_scan, NULL);
+	if (ret < 0) {
+		RTE_LOG(ERR, TIMER, "Cannot set Gatekeeper log timer\n");
```

Comment:
Please use the block macro to log. This comment requires a review of all `RTE_LOG()`s in this patch.